python/ray/air/callbacks/wandb.py

```diff
@@ -148,7 +161,14 @@ def __init__(
     def run(self):
         # Since we're running in a separate process already, use threads.
         os.environ["WANDB_START_METHOD"] = "thread"
-        wandb.init(*self.args, **self.kwargs)
+        run = wandb.init(*self.args, **self.kwargs)
+
+        # Run external hook to process information about wandb run
+        if WANDB_PROCESS_RUN_INFO_HOOK in os.environ:
```

Comment:
do we have test for this code path?

Reply:
I added a unit test for this that only calls `_WandbLoggingProcess.run` by mocking out calls to `wandb`